config/locales/diaspora/en.yml

```diff
@@ -436,22 +436,24 @@ en:
+      whats_in_profile_a: "Your private profile contains your biography, location, gender, and birthday, if you have completed these sections. All this information is optional – it’s up to you whether you provide it. It is displayed on the left column of your profile, under your profile picture."
       who_sees_updates_q: "Who sees updates to my private profile?"
       who_sees_updates_a: "Anyone in your aspects sees changes to your private profile. "
+      public_private_profile_q: "Can I decide to make my private profile... public?"
+      public_private_profile_a: "Yes! We call that the “extended profile“. If your profile represents a software or a foundation for example, you may want to make your (usually) private informations public. To do so, go to the Profile section in the settings and switcht the visibility of your extended profile to “public“."
     public_posts:
       title: "Public posts"
       who_sees_post_q: "When I post something publicly, who can see it?"
       who_sees_post_a: "Anyone using the internet can potentially see a post you mark public, so make sure you really do want your post to be public. It’s a great way of reaching out to the world."
       find_public_post_q: "How can other people find my public post?"
       find_public_post_a: "Your public posts will appear in the streams of anyone following you. If you included #tags in your public post, anyone following those tags will find your post in their streams. Every public post also has a specific URL that anyone can view, even if they’re not logged in – thus public posts may be linked to directly from Twitter, blogs, etc. Public posts may also be indexed by search engines."
       can_comment_reshare_like_q: "Who can comment on, reshare, or like my public post?"
-      can_comment_reshare_like_a: "Any logged-in diaspora* user can comment on, reshare, or like your public post."
+      can_comment_reshare_like_a: "Any logged-in diaspora* user can comment on, reshare, or like your public post, except those you ignored who can't like or comment on your posts."
       see_comment_reshare_like_q: "When I comment on, reshare, or like a public post, who can see it?"
       see_comment_reshare_like_a: "Comments, likes, and reshares of public posts are also public. Any logged-in diaspora* user and anyone else on the internet can see your interactions with a public post."
-      deselect_aspect_posting_q: "What happens when I deselect one or more aspects when making a public post?"
+      deselect_aspect_posting_q: "What happens when I deselect one or more aspects in the left column when making a public post?"
```

Comment:
What happens when I deselect one or more aspects in the **left-hand** column when making a public post?